include/JSystem/J3DGraphAnimator/J3DModel.h

```diff
@@ -128,16 +129,16 @@ class J3DModel {
     Mtx33** getBumpMtxPtrPtr(u32 idx) const { return mpBumpMtxArr[1][idx]; }
     Mtx33* getBumpMtxPtr(u32 idx) const { return mpBumpMtxArr[1][idx][mCurrentViewNo]; }
     void setBaseScale(const Vec& scale) { mBaseScale = scale; }
-    void setUserArea(u32 area) { mUserArea = area; }
-    u32 getUserArea() const { return mUserArea; }
+    void setUserArea(void* area) { mUserArea = area; }
+    void* getUserArea() const { return mUserArea; }
```

Comment:
even though void* makes more sense, the map shows u32 was originally used, so it's probably best to keep it as close as possible